PiwikTracker.php

```diff
@@ -11,7 +11,9 @@
  * @package MatomoTracker
  */
 
-include_once('./MatomoTracker.php');
+if (!class_exists('\MatomoTracker')) {
```

Comment:
Might need to set in both uses the second parameter to `$autoload = false`? Otherwise if there was some autoloader loading the class somehow it might result in duplicate class? Like someone might have auto loader defined for the class to not needing to manually include it etc? Not sure... be also not too important I suppose

Reply:
Having `autoload = true` would automatically load the class if it's loadable using the autoloader (for example with composer autoloader). If it's the case it wouldn't include the `MatomoTracker.php`, so the file would only define the `PiwikTracker` class. Same applies for the class of the other file :thinking:

Reply:
OK 👍 looks all good then